```
$ make build_test
ocamlfind opt -g -package re,re.perl,cmdliner,cucumber,base -linkpkg -cclib -lgherkin -o cucumber_run test/test.ml 
File "test/test.ml", line 68, characters 7-33:
Error: This expression has type unit but an expression was expected of type
         int
make: *** [build_test] Error 2
```

--- a/lib/lib.ml
+++ b/lib/lib.ml
@@ -108,7 +108,7 @@ let manage_command_line cucc tags_str files =
     | None ->
        Tag.list_of_string ""
   in
-  let exit_status = Base.List.fold (Base.List.rev files) ~init:0 ~f:(execute_pickle_lst cucc tags) in
+  let exit_status = Base.List.fold files ~init:0 ~f:(execute_pickle_lst cucc tags) in
   if exit_status = 0 then
     `Ok 0
   else

--- a/test/test.ml
+++ b/test/test.ml
@@ -65,4 +65,4 @@ let bar = [
           
 let _ =
   let _cucc = Base.List.fold bar ~init:Cucumber.Lib.empty ~f:(fun accum stepdef -> stepdef accum) in
-  exit (Cucumber.Lib.execute foo)
+ Cucumber.Lib.execute foo

